src/plugins/graphql/index.ts

```
@@ -0,0 +1,46 @@
+        fastify,
+        isSubscription: false,
+        request,
+        reply,
+      }),
+    graphiql: {
+      enabled: fastify.envConfig.API_IS_GRAPHIQL,
+    },
+    path: "/graphql",
+    schema,
+    subscription: {
+      context: async (socket: any, request: FastifyRequest) =>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)